frontend/packages/dev-console/features/pipeline-Add_pages_for_tekton_trigger

```diff
@@ -0,0 +1,89 @@
+   And User navigate to Pipelines via the tabs on the top of the page
+   Then The Pipelines page contains the Pipelines Name, Last Run, Last Run Status, and Last Started columns
+   And  User navigate to Pipeline Runs via the tabs on the top of the page
+   And The Pipelines Runs page contains the Name, Status, Started, and Duration columns.
+   And User navigate to Pipeline Resources via the tabs on the top of the page
+   And The Pipelines Resources page contains the Name, Created, and Type columns.
+   And User navigate to Conditions via the tabs on the top of the page
+   And The Conditions page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in pipeline page
+   Given User is in Admin view
+   When User clicks pipelines under Pipelines tab
+   And The user click on the Create button from top right corner.
+   Then The main action the user can take on the Pipelines page are create Pipelines, Pipeline Resources and Conditions
+
+@regression
+Scenario: View Pipeline task resources
+   Given User is in Admin view
+   When User clicks Tasks under Pipelines tab
+   And User navigate to Tasks via the tabs on the top of the page
+   Then The Tasks page contains the Name and Created columns.
+   And User navigate to Task Runs via the tabs on the top of the page
+   And The Task Runs page contains the Name, Status, Started, and Duration columns.
+   And User navigate to Cluster Tasks via the tabs on the top of the page
+   And The Cluster Tasks page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in task page
+   Given User is in Admin view
+   When User clicks task under Pipelines tab
+   And The user click on the Create button from top right corner.
+   Then The main action the user can take on the tasks page are create a Task, Task Run and Cluster Tasks
+
+@regression
+Scenario: View Pipeline triggers resources
+   Given User is in Admin view
+   When User clicks Triggers under Pipelines tab
+   And User navigate to Event Listeners via the tabs on the top of the page
+   Then The Event Listeners page contains the Name and Created columns.
+   And User navigate to Trigger Templates via the tabs on the top of the page
+   And The Trigger Templates page contains the Name, Created, and Type columns.
+   And User navigate to Trigger Bindings via the tabs on the top of the page
+   And The Trigger Bindings page contains the Name and Created columns.
+   And User navigate to Cluster Trigger Bindings via the tabs on the top of the page
+   And The Cluster Trigger Bindings page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in Trigger page
+   Given User is in Admin view
+   When User clicks Trigger under Pipelines tab
+   And The user click on the Create button from top right corner.
+   Then The main action the user can take on the Trigger page are create a Event Listener, Trigger Templates, Trigger Binding and Cluster Trigger Binding
```

Comment:
Split it in multiple Then statements

Reply:
For all the statements, start with small letters and not with capitals. 
Eg. **User** should be replaced with **user** if it is at the start of the statement.